index.js

```diff
@@ -12,12 +12,28 @@ if (typeof browser === 'undefined' && !isNodeEnv) {
     browser = chrome
 }
 
+let isEnabled = true;
+
+browser.storage.local.get('enabled', function (data) {
+    isEnabled = data.enabled !== false;
+});
+
+browser.storage.onChanged.addListener((changes, area) => {
+    if (area === 'local' && 'enabled' in changes) {
+        isEnabled = changes.enabled.newValue;
+    }
+});
```

Comment:
Everything seems good. But I don't want to break the current user when upgrade extension.

So I think we should make extension on by default. And if `disabled` value → disable extension.

What do you think @tcortega